DEVELOPMENT.md

```diff
@@ -54,10 +54,32 @@ hatch run test
 ```
 
 ## Run integration tests
+- Pre-requisites:
+  - Developer must have a valid AWS account.
+  - Developer must create a Farm and Queue. This can be done easily via Deadline Cloud's AWS Console quick Farm create workflow. Please copy the Farm resource IDs for the next step. Please copy the Queue's Job Attachment S3 bucket name for the next step.
```

Comment:
super nit

- `IDs` -> `ID`
- It'd be nice to mention the Queue JA bucket root prefix, that's corresponding to `JA_TEST_ROOT_PREFIX` right?

Reply:
How about i change `Please copy the Queue's Job Attachment S3 bucket name and root prefix` for the next step.

Reply:
Looks good!